bitnami/kafka/templates/poddisruptionbudget.yaml

```
@@ -1,6 +1,10 @@
 {{- $replicaCount := int .Values.replicaCount }}
 {{- if and .Values.pdb.create (gt $replicaCount 1) }}
+{{- if .Capabilities.APIVersions.Has "policy/v1/PodDisruptionBudget" }}
+apiVersion: policy/v1
+{{- else }}
 apiVersion: policy/v1beta1
+{{- end }}
```

We have a [named templated](https://github.com/bitnami/charts/blob/master/bitnami/common/templates/_capabilities.tpl#L21) available in the `common` chart that you can use for this:
```suggestion
apiVersion: {{ include "common.capabilities.policy.apiVersion" . }}
```